USER-INTERFACE-INTERFACE.md

```diff
@@ -564,80 +564,53 @@ field will be null or absent.
 ##### Correspondent: Node
 ##### Layout:
 ```
-"payload": {
-    "payableMinimumAmount" = <nonnegative integer>,
-    "payableMaximumAge" = <nonnegative integer>,
-    "receivableMinimumAmount" = <nonnegative integer>,
-    "receivableMaximumAge" = <nonnegative integer>
-}
+"payload": {}
 ```
 ##### Description:
-Requests a financial report from the Node.
-
-In most cases, there will be many records in the database, most of them irrelevant because of amount or age.
-Therefore, when the UI requests a financial report, it should specify minimum amounts and maximum ages. Records
-with amounts smaller than the minimums, or older than the maximums, won't be included in the results, although
-their values will be included in the totals.
-
-This request will result in a cluster of queries to the database, which are quick but not instantaneous,
-especially on old databases that contain lots of records. A UI that makes this request too many times per
-second will perceptibly degrade the performance of the Node.
+Requests financial statistics from the Node.
 
-Amounts are specified in gwei (billions of wei); ages are specified in seconds. Values less than zero or
-greater than 64 bits long will cause undefined behavior.
+This will report back information about Node's performance, that is records of services having been run over time
```

Comment:
Suggested rewording: "This request will report back information about a Node's historical financial operations. This will include both services ordered from other Nodes and services provided for other Nodes, represented as monetary values owed and paid to and from this Node's wallets."